src/main.rs

```diff
@@ -110,6 +105,9 @@ fn main() {
             .arg_from_usage("--dry-run 'Print commands to execute instead of running.'")
             .arg_from_usage("-p, --push 'Push generated documentation to git remote.'")
             .arg_from_usage("-s, --sign 'Sign the git commit.'")
+            .arg_from_usage("-v, --verbose 'Use verbose output.'")
+            .arg_from_usage("-q, --quiet 'Less output printed to stdout.'")
+            .arg_from_usage("--color 'Coloring: auto, always never.'")
```

Comment:
Those description lines are copied across all cargo commands (if you run `--help` on any of the subcommands). Not particularly descriptive, but consistent with what we have elsewhere. What it means is that you can provide either "auto", "always" or "never" as values and the first will determine whether your terminal supports it and the other two will override the detection. Oh, and I forgot a comma.